Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+  return "You are in the right place"
+
+@app.route("/<opp>/<num1>/<num2>")
```

Comment:
instead of calling `int()` everywhere below, you can specify it in the route definition like so:

```python
@app.route("/math/<operation>/<int:num1>/<int:num2>")
```